lib/authentication/view/edit_profile_page.dart

```diff
@@ -203,13 +213,64 @@ class _EditProfilePageState extends State<EditProfilePage> {
     );
   }
 
+  Future<void> getImage() async {
```

Comment:
I don't see the point in having this defined as a method. Just pass it directly to the onTap parameter as a function.